src/deploy/actions.spec.ts

```diff
@@ -50,7 +50,7 @@ describe('Deploy Angular apps', () => {
     });
```

Comment:
I rather have a specific type than `any` as a general rule when I'm coding In Typescript. I want to know the reasons that lead you to use `any`, maybe I am not seeing something.

Reply:
Same as above, I can rewrite it if you think it's better with types.